src/main/java/com/hubspot/jinjava/el/TruthyTypeConverter.java

```diff
@@ -103,7 +104,20 @@ protected String coerceToString(Object value) {
         return enumSet.iterator().next();
       }
     }
-    return super.coerceToEnum(value, type);
+
+    try {
+      return super.coerceToEnum(value, type);
+    } catch (ELException e) {
+      if (value instanceof String) {
```

Comment:
@mattcoley Since jinjava actually uses the `toString` representation when printing enums, `Enum.valueOf` in the regular `coerceToEnum` will not always work. For that reason, I added extra logic here to check the `toString` values of the enums as well.
You can see an example in the `itGetsDictValuesWithEnumKeysUsingToString` test case below.